otter/models/cass.py

```diff
@@ -1163,15 +1168,18 @@ def _assemble_webhook(cass_data):
                                          webhook_id)
             return _assemble_webhook_from_row(cass_data[0])
 
-        query = _cql_view_webhook.format(cf=self.webhooks_table)
-        d = self.connection.execute(query,
-                                    {"tenantId": self.tenant_id,
-                                     "groupId": self.uuid,
-                                     "policyId": policy_id,
-                                     "webhookId": webhook_id},
-                                    DEFAULT_CONSISTENCY)
-        d.addCallback(_assemble_webhook)
-        return d
+        def fetch_webhook(_):
+            query = _cql_view_webhook.format(cf=self.webhooks_table)
+            d = self.connection.execute(query,
+                                        {"tenantId": self.tenant_id,
+                                         "groupId": self.uuid,
+                                         "policyId": policy_id,
+                                         "webhookId": webhook_id},
+                                        DEFAULT_CONSISTENCY)
+            d.addCallback(_assemble_webhook)
+            return d
+
+        return self.get_policy(policy_id).addCallback(fetch_webhook)
```

Comment:
Good idea. I thought doing that can allow this API to work on non-existent policy but that should not happen since corresponding webhook entry itself will not be there. Will change it